src/DssSpell.t.sol

```diff
@@ -2552,23 +2530,12 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(expectedHash, actualHash);
     }
 
-    function getKNCMat() internal returns (uint256 mat) {
-        (, mat) = spotter.ilks("KNC-A");
-    }
-
-    function testKNCOffboarding() public {
+    function testIlkOffboarding() public {
```

Comment:
I'd suggest commenting this test when not used. It's more time for running the tests/CI + it throws a warning do the unused variable.

Reply:
Done in cf225e4.